src/kudu/security/security_flags.cc

```diff
@@ -27,14 +27,14 @@ namespace security {
 // below. The DH AES ciphers are not included since we are not configured to
 // use DH key agreement.
 const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsCiphers =
+                                   "TLS_AES_256_GCM_SHA384:"
+                                   "TLS_CHACHA20_POLY1305_SHA256:"
+                                   "TLS_AES_128_GCM_SHA256:"
+                                   "TLS_AES_128_CCM_8_SHA256:"
```

Comment:
TLS_AES_128_CCM_8_SHA256 - > Don't see this cipher installed in prod hosts.

Reply:
We don't need CCM; it is strictly slower than GCM which has hardware acceleration.

Ideally, applications like this should not be setting ciphers themselves. The default OpenSSL cipher list reads from the system wide OpenSSL configuration file to determine the default ciphers. Sane distributions (including the OpenSSL upstream) configure secure ciphers by default.

But since the existing code does this, we can leave this for now; but please do not add the CCM or TLS 1.3 ciphers (TLS 1.3 ciphers are configured differently).

Reply:
Backed out TLSv1.3 ciphers.